api/tests/unit/domain/validators/user-validator_test.js

```diff
@@ -255,6 +255,36 @@ describe('Unit | Domain | Validators | user-validator', function() {
           expect(errors.invalidAttributes[2]).to.deep.equal(expectedMaxLengthEmailError);
           expect(errors.invalidAttributes[3]).to.deep.equal(expectedPasswordError);
         });
+
+        it('should reject with error on field "mustValidateTermsOfService" when incorrect', async() => {
+          // given
+          const expectedError = {
+            attribute: 'mustValidateTermsOfService',
+            message: '"mustValidateTermsOfService" must be a boolean',
+          };
+
+          user.mustValidateTermsOfService = 'false ,';
+
+          // when
+          const errors = await catchErr(userValidator.validate)({ user });
+          _assertErrorMatchesWithExpectedOne(errors, expectedError);
```

Comment:
On pourrait faire le // then ici
```suggestion

         // then 
          _assertErrorMatchesWithExpectedOne(errors, expectedError);
```

Reply:
Corrigé !